lib/matplotlib/_afm.py

```diff
@@ -376,7 +394,8 @@ def get_str_bbox_and_descent(self, s):
             except KeyError:
                 name = 'question'
                 wx, _, bbox = self._metrics_by_name[name]
-            total_width += wx + self._kern.get((namelast, name), 0)
+            total_width += wx
```

Comment:
I'm uncertain if `WX` is the only thing that can be a float, or that's a parsing bug.